chore(docs): Translation of why-contribute-to-gatsby - es (#11)
gatsbyjs/gatsby-es: docs/contributing/why-contribute-to-gatsby.md

---

> título: Porque Contribuir a Gatsby.js?

Reviewer: ```suggestion
title: ¿Por qué contribuir a Gatsby.js?
```

No cambiamos los nombres de las propiedades o valores del sitio, "title" se queda así 💪

---

Hay muchas razones para contribuir a Gatsby.js y a software de código abierto en general. Aquí están algunos de los beneficios por participar:

> - **Crecimiento de tu comunidad.** Trabajando con Gatsby.js en código abierto puede conectarse con personas que trabajan en cosas similares. Puede obtener ayuda o dar ayuda, e incluso puede hacer un conocido IRL! Esto ha sucedido.

Reviewer: ```suggestion
- **Crecer tu comunidad.** Trabajando con Gatsby.js en código abierto puede conectarte con personas que trabajan en cosas similares. Puedes obtener ayuda o dar ayuda, e incluso puedes hacer un conocido en la vida real! Esto ha sucedido.
```

> - **Mejorar tus habilidades.** Cuando trabajas en un error de Gatsby o un pull request, inevitablemente aprenderás más acerca de cómo funciona algo en el proyecto; ya sea haciendo preguntas y reuniendo información, o profundizando en el código tu mismo. Gatsby.js también ofrece [sesiones de programación en parejas] (/contributing/pair-programming/) a la comunidad, si hay algo en lo que te gustaría trabajar juntos. ¡Definitivamente hay un beneficio en el desarrollo de habilidades para contribuir en OSS!

Reviewer: ```suggestion
- **Mejorar tus habilidades.** Cuando trabajas en un error de Gatsby o un pull request, inevitablemente aprenderás más acerca de cómo funciona algo en el proyecto; ya sea haciendo preguntas y reuniendo información, o profundizando en el código tu mismo. Gatsby.js también ofrece [sesiones de programación en parejas](/contributing/pair-programming/) a la comunidad, si hay algo en lo que te gustaría trabajar juntos. ¡Definitivamente hay un beneficio en el desarrollo de habilidades por contribuir en proyectos de código abierto!
```

> - **Obteniendo un regalo gratis!** Al participar en el repo en Github, puedes ser elegible para el regalo gratuito que el equipo de Gatsby ofrece como una manera de decir gracias. ¡Con 5 contribuciones, tu puedes avanzar a un segundo nivel con aún más opciones de regalos! Aprende más acerca de esto [sweet swag deal](/contributing/contributor-swag/).

Reviewer: ```suggestion
- **Obtener un regalo gratis!** Al participar en el repositorio en Github, puedes ser elegible para un regalo gratuito que el equipo de Gatsby ofrece como una manera de darte las gracias. ¡Con 5 contribuciones, puedes avanzar a un segundo nivel con aún más opciones de regalos! Aprende más acerca de esto en [geniales acuerdos de regalos](/contributing/contributor-swag/).
```

> - **Convirtiéndote en un experto.** Podrías empezar de forma totalmente nueva en un proyecto como Gatsby o React.js, y construyendo cosas y [abriendo errores](/contributing/how-to-file-an-issue/) cuando encuentres un nuevo reto o un error, aprenderás mucho con el tiempo. Mantenerse al día con tu framework favorito a través de su repo de código abierto también puede darte una ventaja en los próximos cambios. Eventualmente, tu podrías encontrar que el conocimiento de dicho proyecto ha crecido tanto que te has convertido en un experto. ¡Celebra actualizando tu curriculum vitae!

Reviewer: ```suggestion
- **Convirtiéndote en un experto.** Podrías empezar de forma totalmente nueva en un proyecto como Gatsby o React.js, y construyendo cosas y [abriendo issues](/contributing/how-to-file-an-issue/) cuando encuentres un nuevo reto o un error, aprenderás mucho con el tiempo. Mantenerte al día con tu framework favorito a través de su repositorio de código abierto también puede darte una ventaja en los próximos cambios. Eventualmente, podrías darte cuenta que tu conocimiento de dicho proyecto ha crecido tanto que te has convertido en un experto. ¡Celebra actualizando tu curriculum vitae!
```

> - **Hablando en conferencias.** Con esas habilidades que te has ganado trabajando en código abierto, ¿por qué no compartir ese conocimiento con el mundo? Las conferencias web en todo el mundo están continuamente buscando oradores y nuevas tomas de temas populares. ¿Construiste algo con Gatsby.js que te entusiasma? ¡La [industria le encantaría saber de ti] (http://weareallaweso.me/)!

Reviewer: ```suggestion
- **Hablando en conferencias.** Con esas habilidades que te has ganado trabajando en código abierto, ¿por qué no compartir ese conocimiento con el mundo? Las conferencias web en todo el mundo están continuamente buscando oradores y nuevas tomas de temas populares. ¿Construiste algo con Gatsby.js que te entusiasma? ¡La [industria le encantaría saber de ti](http://weareallaweso.me/)!
```

Cuando encuentres partes como `[industria le encantaría saber de ti](http://weareallaweso.me/)` no separes el `[]` de los `()` ya que estos juntos son el formato en Markdown para enlaces ✍️

> Hay sin duda más beneficios que los que figuran aquí. ¿Gatsby.js y software de código abierto hacen tu vida mejor de alguna manera? Nos encantaría escucharlo. ¡Contáctenos [en Twitter](https://twitter.com/gatsbyjs)!

Reviewer: ```suggestion
Hay sin duda más beneficios que los que aparecen aquí. ¿Gatsby.js y el software de código abierto hacen tu vida mejor de alguna manera? Nos encantaría escucharlo. ¡Contáctanos [en Twitter](https://twitter.com/gatsbyjs)!
```
